source/dubregistry/repositories/gitlab.d

```diff
@@ -148,7 +148,7 @@ class GitLabRepository : Repository {
 		if (ver.startsWith("~")) ver = ver[1 .. $];
 		else ver = ver;
 		auto venc = () @trusted { return encodeComponent(ver); } ();
-		return m_baseURL.toString()~m_owner~"/"~m_projectPath~"/repository/archive.zip?ref="~venc;
```

Comment:
imo the query parameter "sha" feels like this is passing a wrong argument. Can you add a comment before this line saying that "sha" can be any tag or branch reference too as per GitLab docs? Otherwise it might be questioned in the future

Reply:
Ok, done